resources/views/notifications.blade.php

```diff
@@ -109,7 +109,7 @@
         <button type="button" class="close" data-dismiss="alert">&times;</button>
         <i class="fas fa-exclamation-triangle faa-pulse animated"></i>
         <strong>{{ trans('general.error') }}: </strong>
-        {{ $message }}
+        {!! $message !!}
```

Comment:
This is a no-go, unfortunately. We do have an unescaped notification in that file, but you have to invoke it deliberately. We can never ever trust user input, including asset tags. A bad actor could inject malicious javascript which could hijack cookies, steal sessions, etc. 

If you use the unescaped notification, you MUST make sure that you escape the data before you pass it to the notification. To test this, you can add a new asset (or edit an existing one) with an asset tag of `<script>alert(1)</script>`